README.adoc

```diff
@@ -69,21 +82,27 @@ You can modify (add, modify or delete) registry keys, directories and files like
 
 Registry keys:
 
-    ./malboxes.py registry profile modtype key name value valuetype
+    malboxes registry <profile> <modtype> <key name> <value> <valuetype>
 
-    Ex: ./malboxes registry win10_64_analyst add HKCU:\Software Malboxes IsAwesome String
+Example:
+
+    malboxes registry win10_64_analyst add HKCU:\Software "Malboxes IsAwesome" String
```

Comment:
I'm pretty sure that this isn't the same thing. The first modifies HKCU:\Sofware's Malboxes key with the value IsAwesome as a String. I'm not sure about the behaviour of the second one since we do not provide a value for the "Malboxes IsAwesome" key.